lib/routes/veeam/get.js

```diff
@@ -18,7 +18,7 @@ function getVeeamFile(request, response, bucketMd, log) {
         return responseXMLBody(errors.NoSuchBucket, null, response, log);
     }
     if ('tagging' in request.query) {
-        return respondWithData(request, response, log, request.bucketName,
+        return respondWithData(request, response, log, bucketMd,
```

Comment:
Context for reviewers: passing the whole bucket object instead of the name only will allow to support proxies by returning the right CORS config, API format is not impacted

Reply:
Context: We had an error in local env when retrieving the capacity xml file, so to fix it we sent the bucket object as a parameter in response instead of just the bucket name.